lms/static/sass/course/_info.scss

```diff
@@ -1,11 +1,37 @@
+div.welcome-wrapper {
+  max-width: 1180px;
+  margin: 0 auto;
+
+  .welcome {
+    margin: $baseline;
+    h1 {
+      margin-bottom: 5px;
+      color: $dark-gray1;
+      font-size: 24px;
+    }
+    h2 {
+      color: $dark-gray2;
+      font-size: 14px;
+      text-transform: none;
+    }
+  }
+}
+
 div.info-wrapper {
 
+  background-color: rgb(252, 252, 252);
+
   section.updates {
     @extend .content;
+    @include padding-left($baseline);
     line-height: lh();
 
-    > h1 {
-      @extend .top-header;
+    h1 {
```

Comment:
Unfortunately this markup is whatever the course author decided to write in Studio, so we can only style this per-element.